chapters/packaging.Rmd

```diff
@@ -810,7 +810,7 @@ which we can install with:
 ```
 
 Following the [Python Packaging User Guide][pypi-user-guide],
-and then upload our distributions from the `dist/` folder
+we upload our distributions from the `dist/` folder
```

Comment:
The full sentence reads roughly as:

**Before**
'Following the guide, and then upload distributions, and view the results at \<site\>.'

**After**
'Following the guide, we upload distributions, and view the results at \<site\>.'